Library/.rubocop.yml

```diff
@@ -8,7 +8,17 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+Homebrew/FormulaCop:
```

Comment:
Maybe worth putting this in another directory so it doesn't need to be disabled like this?